### ISSUE TITLE
[Request New Content] Tips for optimizing your labeling operations

### ISSUE THREAD

**Issue by theodu** [kili-technology/kili-blogger-program#10]
### Community vote
👍 Upvote if you like this topic

Thank you.

**Comment by canozcifci** [comment]
1. Introduction --> What is labeling in AI? - Why do we need it?
2. Strategies and practices for optimizing the labeling
3. Building a Better Label Process
4. Practical examples and differences
5. Outro and acknowledgments

**Comment by theodu** [comment]
Hi @canozcifci,
Thank you for your interest!
The outline works for me but can you detail it a bit more with tips that you might address in this article?
Thank you in advance!

Merry Christmas

**Comment by DamianArado** [comment]
Outline:

This article will demonstrate the collection, labeling, and validation of data in order to make it ready to be deployed into production using the TensorFlow Extended (TFX) platform which will be helping us in creating and managing the production pipeline.

Key Objectives:
- ML pipelines in production
- Distinguishing ML modeling and ML in production
- Collecting Data and its importance
- Labeling Data and its importance
- Process Feedback
- Addressing data and concept change by annotating new training data with direct labeling and/or human labeling
- Detecting issues during data validation
- Direct, semi-supervised, weak supervision and active learning methods for labeling data
- Increase the diversity of your training set by data augmentation
- Perform advanced data preparation and transformation on different structured and unstructured data types

The idea is to provide a comprehensive blog according to the aforementioned points. Please let me know what you think. 

Have a nice weekend!

**Comment by canozcifci** [comment]
Hi @theodu 

1. Introduction 
What is labeling in AI? Why do we need it? Create a project on your labeling platform, complete with data uploads and a labeling ontology. Complete any data pre-selection or pre-processing that is required.

2. Strategies and practices for optimizing the labeling
Make a list of labeling policies. Include detailed instructions on how to annotate your data, as well as examples of well and wrongly labeled assets.

3. Building a Better Label Process
Some of the top factors to consider before choosing a data labeling platform;
- The right labeling platform will provide you with all of the tools you'll need to produce high-quality data labels. It's critical to underline that your tool of choice should not only fulfill your current requirements but also be capable of meeting your future requirements.
- Your preferred data platform should include an integrated management system that allows you to manage data, people, and projects all in one location. Project managers can track workers' productivity, develop data labeling workflows, monitor quality assurance, and execute other activities thanks to their resilience.
- Your label platform should have a quality assurance mechanism that allows you to have complete control over the output quality. Remember that the quality of your labeled data determines the success of your machine learning models.
- You'll do better with a data labeling platform that has a responsive support team, as always. The team should be prepared to address issues as soon as they arise, ensuring that your workflow is not disrupted.

4. Practical examples and differences
Here are some labeling tools;
- Scale.ai
- Labelbox
- AWS SageMaker Ground Truth
- Cord
- labelImg
- Image Labelling Software of Kili Technology

5. Outro and acknowledgments
Conclusion, references, special thanks

Merry Christmas :+1:

**Comment by theodu** [comment]
Thank you for your outlines! The aim of this article would be more to emphasize the labeling operations, data preparation, data augmentation, etc would be tackled in another article @DamianArado.

@canozcifci I like your outline. It shows what is important while annotating and how annotation platforms provide an all-in-one solution to handle operations. We could see how the title could be adapted to your content.

I find it nice for the 4th part to be objective and to talk about some concurrent but I would have to check with the marketing team if they have any constraints. I keep you in touch with this.

 @canozcifci I assign you to it, you can begin to work on it!

**Comment by canozcifci** [comment]
Hi @theodu

Here is my first draft of the article.
https://docs.google.com/document/d/1ovKSU3-O0SSM6IX2TjnutI-OYMHcQ1UxVIIOeIKyj5k/edit?usp=sharing

Looking forward to hearing your feedback. Contact me via cangoksu191@gmail.com

Have a nice day.
Can

**Comment by theodu** [comment]
Hi @canozcifci 
thank you! we will have a look asap and I get back to you to give feedback ;-)

**Comment by theodu** [comment]
Hi @canozcifci 
I went through your article. I noticed some very good parts and some bad parts. then I saw plagiarism and now I understand.
Should I mention that by doing plagiarism you, and us actually, risk very serious legal consequences?
I have to unassign you from this article.
I hope that you will understand that this is unacceptable and that you legally risk a lot by doing so.
